src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirDataStore.cs

```diff
@@ -441,6 +443,8 @@ public async Task<ResourceWrapper> GetAsync(ResourceKey key, CancellationToken c
                             rawResource = await _compressedRawResourceConverter.ReadCompressedRawResource(rawResourceStream);
                         }
 
+                        _logger.LogInformation($"{nameof(resourceSurrogateId)}: {resourceSurrogateId}; {nameof(rawResource)} length: {rawResource.Length}");
```

Comment:
This looks similar to the log in the Search Service. Do you also want the resource type in this information?

Reply:
Good catch, I've added the ResourceType to the output.